### PR TITLE
Seedr

### PR DESCRIPTION
**Pull request by Si3rr4wow**

### CODE REVIEW

File: src/seedr/products/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  it('seeds mongo and elastic', async () => {
+    await seedProducts(10)
+    expect(seedMongoProducts).toHaveBeenCalledTimes(1)
+    expect(seedElasticProducts).toHaveBeenCalledTimes(1)
+  })
+
+  describe('if seeding mongo fails', () => {
+    it('deletes everything', async () => {
+      ;(seedMongoProducts as jest.Mock).mockRejectedValueOnce('error')
```
Comment (Xfarya) [inline src/seedr/products/index.test.ts:58]: these semi-colons are auto-formatted?

File: src/seedr/utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    // followed by punctuation
+    // followed by whitespace
+    // ends with punctuation
+    // eg '[I saw one of the babies. It looked at me.]' one match
+    // 'dinosaurs are old' no matches
+    // 'Dinosaurs are old' no matches
+    // 'Dinosaurs are old.' no matches
+    // '[Dinosaurs are old. They are all kill' no matches
+    // '[Dinosaurs are old. They are all kill.]' one match
+    const sentenceRegex = /[A-Z][a-z\s]+[.|?|!]\s[A-Z][a-z\s]+[.|?|!]$/g
```
Comment (Xfarya) [inline src/seedr/utils.test.ts:48]: is this regex working?

File: src/seedr/products/mongo.test.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import {
+  deleteAllProducts,
+  formatProductsForBulkInsert,
+  seedProducts,
+} from './mongo'
+import { randomProducts } from './randomProducts'
+import * as db from '../../__test__/setup/db'
+import { productModel } from '../../model/Product'
+import { selectRandom } from '../utils'
```
Comment (Xfarya) [inline src/seedr/products/mongo.test.ts]: can be removed

File: src/seedr/products/index.test.ts (hunk 1 above)
Comment (Si3rr4wow) [reply src/seedr/products/index.test.ts:58]: yeah no-semi has weird consequences

File: src/seedr/products/mongo.test.ts (hunk 3 above)
Comment (Si3rr4wow) [reply src/seedr/products/mongo.test.ts]: good catch

File: src/seedr/utils.test.ts (hunk 2 above)
Comment (Si3rr4wow) [reply src/seedr/utils.test.ts:48]: yup, ish

**Pull request merged**